### PR TITLE
Dockerfile: adding jsonc and mcrypt by default

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -38,6 +38,8 @@ RUN apt-get update && \
         php5-curl=5.6.6+dfsg-1+deb.sury.org~trusty+1 \
         php5-gearman=1.1.2-1+deb.sury.org~trusty+2 \
         php5-memcache=3.0.8-5+deb.sury.org~trusty+1 \
+        php5-mcrypt=5.6.6+dfsg-1+deb.sury.org~trusty+1 \
+        php5-json=1.3.6-1+deb.sury.org~trusty+2 \
```
Comment (ingluisjimenez) [inline Dockerfile:42]: is this the installed version of 5.6 bodegas?

On 5.4 bodegas, it is 1.2.1.

**Comment by jimdelois** [comment]
:+1:

File: Dockerfile (hunk 1 above)
Comment (bryanlatten) [inline Dockerfile:42]: @ingluisjimenez yes

# Installed packages, channel pecl.php.net:

Package  Version State
gearman  1.0.3   stable
igbinary 1.2.1   stable
imagick  3.1.2   stable
`jsonc    1.3.6   stable`
memcache 3.0.8   beta
mongo    1.2.12  stable
xdebug   2.2.5   stable
zip      1.12.4  stable
[root@dev45 core]#

**Comment by ingluisjimenez** [comment]
:+1:

**Comment by chrishenry** [comment]
:+1:

**Pull request merged**